sensum-pwa/src/app/App.res

```diff
@@ -44,6 +47,7 @@ let make = () => {
     Promise.resolve()
   })
   ->ignore
-
+  
+  State.Configuration.loadNetworkHook()
```

Comment:
start app by loading settings from local storage

Reply:
This file feels weird jaja, guess it's because of the syntax, dunno

Reply:
Don't worry, the file still has a lot of things that are wrong and will be removed. Like the `Sensations` and blockchain related stuff